src/main/java/com/arpnetworking/clusteraggregator/GuiceModule.java

```diff
@@ -344,8 +343,9 @@ private PeriodicMetrics providePeriodicMetrics(final MetricsFactory metricsFacto
     @Provides
     @Singleton
     @SuppressFBWarnings("UPM_UNCALLED_PRIVATE_METHOD") // Invoked reflectively by Guice
-    private AggMessageExtractor provideExtractor() {
-        return new AggMessageExtractor();
+    private AggMessageExtractor provideExtractor(
+            @Named("reaggregation-dimensions") final ImmutableSet<String> reaggregationDimensions) {
```

Comment:
It doesn't look like you're providing this ImmutableSet anywhere.

Reply:
It was added in #85 and can be seen [here](https://github.com/ArpNetworking/metrics-cluster-aggregator/blame/40a289b9b09414ef7afbda88797d069b94be03ac/src/main/java/com/arpnetworking/clusteraggregator/GuiceModule.java#L374).